Software/remarkable.yml

```diff
@@ -0,0 +1,21 @@
+Name: remarkable
+Description: Desktop app for the reMarkable tablet.
+Grade: Gold
+Arch: win64
+
+Dependencies:
+- d3dcompiler_43
+
+Parameters: {}
+
+Executable:
+  name: reMarkable
+  icon: remarkable.ico
+  file: reMarkable.exe
+  path: Program Files (x86)/reMarkable/reMarkable.exe
```

Comment:
I've tried both with and without the executable as part of the path here.
The [maintainers.usebottles.com documentation](https://maintainers.usebottles.com/installers/structure/) seem to indicate that the executable file should be part of the path, but the [guidelines file in this repo](https://github.com/bottlesdevs/programs/blob/main/GUIDELINES.md) specifies `without the executable itself`.

Other installers seem to do both, so I don't really think is the issue that makes the folder of the executable open instead of the program itself.